src/components/Drawer.js

```diff
@@ -152,11 +159,11 @@ const Drawer = React.createClass({
         backgroundColor: StyleConstants.Colors.PORCELAIN,
         boxShadow: StyleConstants.ShadowHigh,
 
-        '@media (max-width: 750px)': {
+        [`@media (max-width: ${this.props.breakPoints.medium}px)`]: {
```

Comment:
I made the realization of this ^ on the drive home (second best place to do thinking). I like using string interpolation and I think this is a really creative implementation of it!